packages/react-aria-components/docs/react-aria-components.mdx

```diff
@@ -323,8 +323,6 @@ import {Select, SelectValue, Label, Button, Popover, ListBox, Item} from 'react-
 
 </details>
 
-If you're using SSR via a framework like Next.js or Remix, you'll also need to wrap your application in an `<SSRProvider>`. See the [Server Side Rendering](ssr.html) docs for more details.
```

Comment:
Should we leave this but mention this is true when using react 16 or 17?

Reply:
idk maybe. I figured most people using RAC will be using modern React, and if not they can find info on the SSR docs page. Putting lots of caveats all over the place might be more confusing. Not sure.